src/index.js

```diff
@@ -42,6 +44,29 @@ function isCID (hash) {
   }
 }
 
+function isMultiaddr (input) {
+  if (!input) return false
+  if (isString(input) || input instanceof Buffer) {
+    try {
+      new Multiaddr(input) // eslint-disable-line no-new
+      return true
+    } catch (e) {
+      return false
+    }
+  }
+  if (Multiaddr.isMultiaddr(input)) return true
+  return false
+}
+
+function isPeerMultiaddr (input) {
+  if (!isMultiaddr(input)) return false
+  if (input instanceof Buffer) {
+    // mafmt does not support Buffer input
+    input = new Multiaddr(input)
+  }
+  return mafmt.IPFS.matches(input)
```

Comment:
Again, I'd either pass `mafmt` a `Multiaddr` for ALL `input`s or change `mafmt` to accept buffers also (I mean that sounds reasonable right?)

Reply:
In the spirit of this lib doing as little as possible, I fixed Buffer support in `mafmt`: https://github.com/multiformats/js-mafmt/pull/39 Will switch this PR to that when merged & released.